app/controllers/content_items_controller.rb

```diff
@@ -56,4 +63,16 @@ def error_403(exception)
   def error_notfound
     render plain: 'Not found', status: :not_found
   end
+
+  def ab_testing_group
```

Comment:
Could do, although we'd still need a method here to check if the content is tagged to a taxon. Where's the best place for such a method to live?